frontend/src/utilities/valueUnits.ts

```diff
@@ -22,8 +22,9 @@ export const CPU_UNITS: UnitOption[] = [
   { name: 'Milicores', unit: 'm', weight: 1 },
 ];
 export const MEMORY_UNITS_FOR_SELECTION: UnitOption[] = [
-  { name: 'GiB', unit: 'Gi', weight: 1024 },
-  { name: 'MiB', unit: 'Mi', weight: 1 },
+  { name: 'TiB', unit: 'Ti', weight: 1024 ** 4 },
+  { name: 'GiB', unit: 'Gi', weight: 1024 ** 3 },
+  { name: 'MiB', unit: 'Mi', weight: 1024 ** 2 },
```

Comment:
It looks good!

I was also testing putting in 1024Ti and it does keep going with the higher versions. So then it would be saved at `1Pi` and opening it would become `1Ti`. So i suppose we could add the the other 2 to match the parsing logic?

```suggestion
  { name: 'EiB', unit: 'Ti', weight: 1024 ** 6 },
  { name: 'PiB', unit: 'Ti', weight: 1024 ** 5 },
  { name: 'TiB', unit: 'Ti', weight: 1024 ** 4 },
  { name: 'GiB', unit: 'Gi', weight: 1024 ** 3 },
  { name: 'MiB', unit: 'Mi', weight: 1024 ** 2 },
```

Reply:
Looks good! 
![image](https://github.com/user-attachments/assets/b3f32458-e69b-4b9e-aa61-009637ce9acf)